tools/solidity-coverage-example/coverage.json

```diff
@@ -0,0 +1 @@
statementMap":{"1":{"start":{"line":14,"column":8},"end":{"line":14,"column":327}},"2":{"start":{"line":27,"column":8},"end":{"line":27,"column":71}},"3":{"start":{"line":28,"column":8},"end":{"line":28,"column":59}},"4":{"start":{"line":30,"column":8},"end":{"line":30,"column":63}},"5":{"start":{"line":32,"column":8},"end":{"line":32,"column":44}}},"branchMap":{"1":{"line":14,"type":"if","locations":[{"start":{"line":14,"column":8},"end":{"line":14,"column":8}},{"start":{"line":14,"column":8},"end":{"line":14,"column":8}}]},"2":{"line":27,"type":"if","locations":[{"start":{"line":27,"column":8},"end":{"line":27,"column":8}},{"start":{"line":27,"column":8},"end":{"line":27,"column":8}}]},"3":{"line":28,"type":"if","locations":[{"start":{"line":28,"column":8},"end":{"line":28,"column":8}},{"start":{"line":28,"column":8},"end":{"line":28,"column":8}}]}}},"contracts/PrngSystemContract.sol":{"l":{"13":1,"15":1,"16":1,"17":0},"path":"/Users/stefanstefanov/Projects/hedera-json-rpc-relay/tools/solidity-coverage-example/contracts/PrngSystemContract.sol","s":{"1":1,"2":1,"3":0},"b":{"1":[1,0]},"f":{"1":1},"fnMap":{"1":{"name":"getPseudorandomSeed","line":12,"loc":{"start":{"line":12,"column":4},"end":{"line":18,"column":4}}}},"statementMap":{"1":{"start":{"line":13,"column":8},"end":{"line":13,"column":474}},"2":{"start":{"line":15,"column":8},"end":{"line":15,"column":50}},"3":{"start":{"line":17,"column":8},"end":{"line":17,"column":40}}},"branchMap":{"1":{"line":15,"type":"if","locations":[{"start":{"line":15,"column":8},"end":{"line":15,"column":8}},{"start":{"line":15,"column":8},"end":{"line":15,"column":8}}]}}},"contracts/TokenSend.sol":{"l":{"15":1,"19":1,"20":1,"26":1,"27":1,"28":1,"29":1,"30":1,"32":1,"36":1,"42":1},"path":"/Users/stefanstefanov/Projects/hedera-json-rpc-relay/tools/solidity-coverage-example/contracts/TokenSend.sol","s":{"1":1,"2":1,"3":1,"4":1,"5":1,"6":1,"7":1},"b":{},"f":{"1":1,"2":1,"3":1,"4":1,"5":1},"fnMap":{"1":{"name":"constructor","line":14,"loc":{"start":{"line":14,"column":4},"end":{"line":16,"column":4}}},"2":{"name":"loadFunds","line":18,"loc":{"start":{"line":18,"column":4},"end":{"line":21,"column":4}}},"3":{"name":"createTransferList","line":23,"loc":{"start":{"line":23,"column":4},"end":{"line":33,"column":4}}},"4":{"name":"storeAmount","line":35,"loc":{"start":{"line":35,"column":4},"end":{"line":37,"column":4}}},"5":{"name":"getAmount","line":39,"loc":{"start":{"line":39,"column":4},"end":{"line":43,"column":4}}}},"statementMap":{"1":{"start":{"line":19,"column":8},"end":{"line":19,"column":102}},"2":{"start":{"line":20,"column":8},"end":{"line":20,"column":40}},"3":{"start":{"line":26,"column":8},"end":{"line":26,"column":131}},"4":{"start":{"line":27,"column":8},"end":{"line":27,"column":107}},"5":{"start":{"line":28,"column":8},"end":{"line":28,"column":113}},"6":{"start":{"line":32,"column":8},"end":{"line":32,"column":42}},"7":{"start":{"line":42,"column":8},"end":{"line":42,"column":27}}},"branchMap":{}}}
```

Comment:
Q: Why does this file have your filepath?
Does it not pick up relative paths? We should update that

Reply:
Now that I look at it again ... Arguably we should not commit this file at all. 
It's the output of the code coverage tool itself. Seems like committing the output of the TypeScript transpilation.

Reply:
Great, yeah let's remove it.